### PR TITLE
Create report view for admins

### LINKED ISSUE #12: Create a report view for admins
# REQUIREMENTS
Create with CSS and HTML the admin view that will be use to show the reports

# DESIGN
## Proposal design
image

### PR DESCRIPTION
**Pull request by DiegoSteve**
# Description

-  What?: 

  Added style to report view

-  Why?:

Being able to have a better interface for administrators and see the employee report

  ## Screenshots
Captura de Pantalla 2022-03-23 a la(s) 18 05 35

### CODE REVIEW

File: app/assets/stylesheets/sass/views/attendance/index.css.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+.sub-container {
+  height: 100%;
+  margin: 5% 5%;
+  width: 80%;
+ 
+  .search-row {
+    display: flex;
+    justify-content: space-between;
+     .month-option{
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/views/attendance/index.css.scss]: Add a extra space between your class and your bracket, and add a extra space between `.search-row` and `.month-option`

File: app/assets/stylesheets/sass/views/attendance/index.css.scss [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        display: flex;
+        font-size: 18px;
+        height: 2rem;
+        justify-content: center;
+        width: 33%;
+
+        select {
+        background-color: $normal-cyan;
+        height: 20px;
+        width: 30px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/views/attendance/index.css.scss:23]: Fix the identation in this block

File: app/assets/stylesheets/sass/views/attendance/index.css.scss [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  p {
+    padding: 0 20px;
+  }
+}
+
+.section {
+  width: 33%;
+}
+
+.father-table{
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/views/attendance/index.css.scss]: Add a extra space between your class and your bracket

File: app/assets/stylesheets/sass/views/attendance/index.css.scss [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        justify-content: center;
+        width: 33%;
+
+        select {
+        background-color: $normal-cyan;
+        height: 20px;
+        width: 30px;
+        }
+      }      
+  }
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/views/attendance/index.css.scss]: Fix this trouble of identation

File: app/assets/stylesheets/sass/views/attendance/index.css.scss [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    background-color: $very-dark-cyan;
+    border-radius: 15px;
+    color: white;
+    font-size: 15px;
+    margin-left: 100px;
+    padding-left: 5px;
+    width: 30%;
+  }
+}
+
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/views/attendance/index.css.scss:144]: Add in your VS Code this [extension](https://marketplace.visualstudio.com/items?itemName=riccardoNovaglia.missinglineendoffile)

**Review by JuanequeX — DISMISSED** [review]

**Review by YilKanda — COMMENTED** [review]
Please resolve the observations made by @JuanequeX

File: app/assets/stylesheets/sass/views/attendance/index.css.scss (hunk 1 above)
Comment (DiegoSteve) [reply app/assets/stylesheets/sass/views/attendance/index.css.scss]: Thanks, Done!

File: app/assets/stylesheets/sass/views/attendance/index.css.scss (hunk 2 above)
Comment (DiegoSteve) [reply app/assets/stylesheets/sass/views/attendance/index.css.scss:23]: Thanks, Done!

File: app/assets/stylesheets/sass/views/attendance/index.css.scss (hunk 3 above)
Comment (DiegoSteve) [reply app/assets/stylesheets/sass/views/attendance/index.css.scss]: Thanks, Done!

File: app/assets/stylesheets/sass/views/attendance/index.css.scss (hunk 4 above)
Comment (DiegoSteve) [reply app/assets/stylesheets/sass/views/attendance/index.css.scss]: Done!

File: app/assets/stylesheets/sass/views/attendance/index.css.scss (hunk 5 above)
Comment (DiegoSteve) [reply app/assets/stylesheets/sass/views/attendance/index.css.scss:144]: Done!

**Review by YilKanda — APPROVED** [review]
LGTM!

File: app/views/attendances/_attendance.html.erb [hunk 6]
```diff
@@ -1,17 +1,10 @@
...
-    <%= attendance.employee_id %>
-  </p>
 
+<tbody>
+    <tr>
+      <td><%= attendance.check_in %></td>
+      <td><%= attendance.check_out %></td>
+      <td><%= attendance.employee_id  %></td>
+    </tr>
+  </tbody>
```
Comment (diegoOG09) [inline app/views/attendances/_attendance.html.erb:9]: Check the identation here, is not the appropiate

**Review by diegoOG09 — APPROVED** [review]
LGTM

**Review by dannharo — APPROVED** [review]
LGTM, just check Diego's Comments

**Review by EliasUR — APPROVED** [review]
LGTM

**Pull request merged**